tests/python/mkl/test_mkldnn.py

```diff
@@ -233,7 +233,7 @@ def check_batchnorm_training(stype):
                            mx.nd.array(beta).tostype(stype)]
             mean_std = [mx.nd.array(rolling_mean).tostype(stype), mx.nd.array(rolling_std).tostype(stype)]
 
-            test = mx.symbol.BatchNorm(data, fix_gamma=True)
+            test = mx.symbol.BatchNorm(data, fix_gamma=False)
```

Comment:
Could you elaborate why you are modifying a test case?

Reply:
I thought we had the conversation in Lai's PR yesterday, and I've created the related issue #11647

Reply:
The purpose of this mkl test is to test that we can fallback to sparse matrices correctly when using MKLDNN under the legal cases for sparse matrices. The logics of exclusion of illegal cases for sparse matrices(fix_gamma=True), which is shared by both USE_MKLDNN=1 and USE_MKLDNN=0 situtaions, were already tested by the test_batchnorm_fallback test.

Reply:
Ah I see, sorry, got a lot of PRs swirling around my head. Let me rephrase to ensure I got it right: This means that the test case with gamma=True is basically invalid until that issue has been resolved and that's why you are changing it, correct?
If yes, we're good to go. Sorry for the caused inconvenience.

Reply:
Yes you got it, that's why we're only testing the necessary legal case here. Please merge this once you feel good to do so, thanks!

Reply:
I'd like to request the review from another committer to assess the backend changes if you don't mind.

Reply:
No problem. @zheng-da @eric-haibin-lin

Reply:
BTW I think @anirudh2290 already approved the previous #11631